Formula/cronolog.rb

```diff
@@ -22,10 +22,6 @@ class Cronolog < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5b7e0546314cd92d44b009de1894f4739f75865da5f95d425e305cf6ffeac10d"
   end
 
-  # Upstream is gone.
```

Comment:
This is a reason for deprecation

Reply:
If it still works and analytics show it's still used: let's not unless we have a new deprecation state that doesn't get disabled/removed.